docs/ops.html

```diff
@@ -3971,7 +3971,10 @@ <h3>Enter Migration Mode on the Brokers</h3>
 
 # KRaft controller quorum configuration
 controller.quorum.voters=3000@localhost:9093
-controller.listener.names=CONTROLLER</code></pre>
+controller.listener.names=CONTROLLER
+
+# If the ZK broker is using a generated broker ID, disable broker ID generation
+broker.id.generation.enable=false</code></pre>
```

Comment:
I think we should add more context here. 

Questions:
1. if we don't add this line: `broker.id.generation.enable=false`, will it always fail?
2. If user needs to enable `broker.id.generation.enable` in their cluster because it works for years, what could they do?

Reply:
I see, I will figure out how to clarify these two question. 😃

Reply:
@showuon that is good question. maybe we can describe two solutions for users:

1. `broker.id.generation.enable=false`. in migration, all brokers should have unique id already so it should be fine. 
2. increase `reserved.broker.max.id`. users need to know the max node id in the cluster

Reply:
>If we don't add this line: broker.id.generation.enable=false, will it always fail?

If  `broker.id.generation.enable=true` and the user does not set `reserved.broker.max.id` to a value greater than -1, it will definitely fail; conversely, `broker.id.generation.enable=false` will make `reserved.broker.max.id` ineffective, so it will definitely not fail.

>If the user needs to enable broker.id.generation.enable in their cluster because it has been working for years, what could they do?

It is recommended that the user adjusts `reserved.broker.max.id` to a sufficiently large value to facilitate future migrations.

Reply:
oh, as @chia7712 said, maybe the first question won't be a concern ?